Nos/Views/NoteTextEditor.swift

```diff
@@ -74,6 +75,7 @@ struct NoteTextEditor: View {
     }
     
     private func insertMention(at offset: Int, author: Author) {
+        Log.error("test")
```

Comment:
This sounds like a log you used while debugging

Reply:
oops, you are right.